Assets/Group1/Scripts/Game.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+using System.Collections.Generic;
+
+public class Game : MonoBehaviour
+{
+    [SerializeField] private Player _player;
+    [SerializeField] private List<Enemy> _enemies;
+
+    public UnityAction AllEnemiesDead;
+
+    private void Start()
+    {
+        CheckActiveEnemies();
+
+        foreach (var enemy in _enemies)
+        {
+            enemy.Init(_player);
+        }
+    }
+
+    private void OnEnable()
+    {
+        foreach (var enemy in _enemies)
+        {
+            enemy.EnemyDead += OnEnemyDied;

```

Comment:
https://trello.com/c/QAQLkgbR/7-%D0%B3%D0%B4%D0%B5-%D0%BB%D1%83%D1%87%D1%88%D0%B5-%D0%BF%D0%BE%D0%B4%D0%BF%D0%B8%D1%81%D1%8B%D0%B2%D0%B0%D1%82%D1%8C%D1%81%D1%8F